client/src/main/java/org/apache/rocketmq/client/common/ThreadLocalIndex.java

```diff
@@ -31,7 +32,7 @@ public int incrementAndGet() {
         }
 
         this.threadLocalIndex.set(++index);
-        return Math.abs(index);
+        return Math.abs(index & POSITIVE_MASK);
```

Comment:
Awesome bit operation, but it seems that the Math.abs isn't needed anymore. correct me if I am wrong!